### ISSUE TITLE
Reproducing figure 3 plots

### ISSUE THREAD

**Issue by ShailenSmith** [Pillowkoh/RoLSS#2]
Hi,

In Figure 3 of the RoLSS paper (attached below), how did you achieve the RoLSS models at different accuracy levels? Are they the same model, just taken from different points in training?

Also, I was trying to train some RoLSS ResNet-152s on FaceScrub, and I was noticing that sometimes it took a while for the training to 'get started', e.g. the train accuracy not increasing above 1% for the first 10 epochs or so. Did you have any issues with this?

<img width="1269" height="912" alt="Image" src="https://github.com/user-attachments/assets/b3d6bb52-272f-4fab-b98a-dec2a87d8d1e" />

**Comment by Pillowkoh** [comment]
Hi @ShailenSmith, thank you for reaching out. For these models, each checkpoint refers to the same model at different juncture of the training epochs with different natural accuracy scores. 

For the second issue, yes the model does seem to be stuck for the first few epochs for me as well. However, once the model goes beyond 2% training accuracy, it will start to train as per normal.

**Comment by ShailenSmith** [comment]
Hi @Pillowkoh, thanks for your response. One more question -- did you apply any learning rate scheduler, for example, multiplying by a factor of 0.1 at epoch 80, or something similar?

**Comment by ShailenSmith** [comment]
Hi @Pillowkoh, just wanted to follow up on this -- I see in the paper you use MultiStepLR, but at which epoch did you set the milestones?
